cmd/tuf/app/key-pop-verify.go

```diff
@@ -70,6 +98,63 @@ func KeyPOPVerify() *ffcli.Command {
 	}
 }
 
+func GetKeyIDForRole(directory, role string) (string, error) {
+	store := tuf.FileSystemStore(directory, nil)
+	signed, err := repo.GetSignedMeta(store, targetsMeta)
+	if err != nil {
+		return "", err
+	}
+	meta, err := repo.GetMetaFromStore(signed.Signed, targetsMeta)
+	if err != nil {
+		return "", err
+	}
+	targets := meta.(*data.Targets)
+
+	for _, delegatedRole := range targets.Delegations.Roles {
+		if delegatedRole.Name != role {
+			continue
+		}
+
+		if len(delegatedRole.KeyIDs) != 1 {
```

Comment:
this is a current limitation that we only expect to delegate to one key ID right? maybe comment here

Reply:
Yes, I made this choice as it ignores the issue of matching which signature (POP) belongs to each key. So if a delegation would use multiple keys, we would need to run POP verify for each key, and provide the `keyid` instead of the role name. 

The utility scripts included now don't support that functionality as of now, added this tracking isse https://github.com/sigstore/root-signing/issues/707

Reply:
Sounds good! Let's definitely table it and keep as a tracking issue - for multiple keys / delegation there would need to be some shift of flow (e.g. one commit on the PR per added key ID / POP, or something like that). Not necessary for now.